torch/csrc/distributed/rpc/testing/init.cpp

```diff
@@ -0,0 +1,118 @@
+#include <c10d/ProcessGroup.hpp>
+#include <pybind11/chrono.h>
+#include <pybind11/stl.h>
+#include <torch/csrc/distributed/rpc/process_group_agent.h>
+#include <torch/csrc/distributed/rpc/rpc_agent.h>
+#include <torch/csrc/distributed/rpc/testing/faulty_process_group_agent.h>
+#include <torch/csrc/python_headers.h>
+#include <torch/csrc/utils/pybind.h>
+
+template <typename T>
+using shared_ptr_class_ = py::class_<T, std::shared_ptr<T>>;
```

Comment:
nit: other init.cpp files put this inside the inner anonymous namespace. We can do the same to stay consistent